### PR TITLE
improving test case name in global navigation on mercury

### PR DESCRIPTION
**Pull request by justnpT**
I analyzed the test cases and improved their names as part of https://wikia-inc.atlassian.net/browse/XW-980

### CODE REVIEW

**Comment by vforge** [comment]
:+1:

File: src/test/java/com/wikia/webdriver/testcases/mercurytests/NavigationTest.java [hunk 1]
```diff
@@ -31,7 +31,7 @@ private void init() {
   }
 
   @Test(groups = "mercury-navigation-001")
```
Comment (bognix) [inline src/test/java/com/wikia/webdriver/testcases/mercurytests/NavigationTest.java]: consider renaming groups to something more descriptive so when displayed on dashboard or in Jenkins there is more context available to the reader

Comment (justnpT) [inline src/test/java/com/wikia/webdriver/testcases/mercurytests/NavigationTest.java]: done :+1:

**Comment by justnpT** [comment]
please don't merge until lgtm from @tomasznapieralski 
We have to update the Jenkins group names

**Comment by tomasznapieralski** [comment]
We shouldn't remove `navigation-` prefix, because while running all tests we won't know what was tested.
Please restore prefix, we can drop numeration but we can't drop this prefix.

**Comment by justnpT** [comment]
@tomasznapieralski Done

**Comment by tomasznapieralski** [comment]
Thanks @justnpT good idea to add this prefix to only group name.
I will update groups on Jenkins

**Comment by tomasznapieralski** [comment]
damit `mercury-` prefix shoud be also applied, i will update that

**Comment by tomasznapieralski** [comment]
done, prefix added, tests updated on jenkins

**Pull request merged**